cli/lsp/documents.rs

```diff
@@ -720,13 +723,13 @@ impl RedirectResolver {
 
 #[derive(Debug, Default)]
 struct FileSystemDocuments {
-  docs: HashMap<ModuleSpecifier, Arc<Document>>,
-  dirty: bool,
+  docs: DashMap<ModuleSpecifier, Arc<Document>>,
+  dirty: AtomicBool,
```

Comment:
Probably not a big deal, but I think this could be a `RefCell<bool>`?

Reply:
This is shared by multiple threads.